src/System.Net.Security/src/System/Net/SecureProtocols/_SslState.cs

```diff
@@ -117,25 +117,17 @@ internal void ValidateCreateContext(bool isServer, string targetHost, SslProtoco
             if (targetHost == null)
             {
                 throw new ArgumentNullException("targetHost");
-            }
-            
-            if (isServer)
-            {
-                enabledSslProtocols &= (SslProtocols)Interop.SChannel.ServerProtocolMask;
```

Comment:
It looks like it's in /src/System.Net.Security/src/System/Net/Windows/SSPISecureChannelType.cs `CreateSecureCredential`